docs/Installation.md

```diff
@@ -58,3 +64,5 @@ as well as the supported versions of Python:
 | `2.4.0`   | `2022.1.0` | `1.12.1` | `1.12.0` | `2.8.2`    | `3.8`  |
 
 > (*) Python 3.9 or higher is required for TensorFlow 2.15.1
```

Comment:
I suppose we forgot to remove it together with python 3.8. Let's remove it now

Reply:
It will still actual while we have `3.8*` in corresponding versions.
In some time table will be too long, and we will remove old rows.

Reply:
Ok, you mean it's actual for old version. Let's keep it then.